catalog/app/containers/Bucket/Meta.tsx

```diff
@@ -24,54 +24,42 @@ const errorHandler = (error: Error) => (
 
 const noop = () => null
 
-const useHeadCellStyles = M.makeStyles((t) => ({
-  root: {
-    paddingTop: t.spacing(1),
-    verticalAlign: 'top',
-    whiteSpace: 'nowrap',
-  },
-}))
-
-interface HeadCellProps {
-  children: React.ReactNode
-  className?: string
-  title: string
-}
-
-const HeadCell = ({ className, children, title }: HeadCellProps) => {
-  const classes = useHeadCellStyles()
-  return (
-    <M.TableCell className={cx(classes.root, className)} component="th" scope="row">
-      <M.Tooltip title={title}>
-        <span>{children}</span>
-      </M.Tooltip>
-    </M.TableCell>
-  )
-}
-
-const usePackageMetaStyles = M.makeStyles({
+const usePackageMetaStyles = M.makeStyles((t) => ({
   headCell: {
+    paddingTop: t.spacing(0.25),
     width: '137px',
   },
+  cellExpanded: {
```

Comment:
does this mean that the json block is displayed inline while not expanded, and after expansion it jumps to the new line?

Reply:
Yes, that was the main idea. Does it bother you?

It doesn't look nice at this moment, but after https://github.com/quiltdata/quilt/pull/3249 it will look nicer

Reply:
yeah, this kind of jump seems like a bad UX to me. i'd rather always display it on its own line or keep it inline, it doesn't matter that much imo, but the jump bothers me

Reply:
I agree that it doesn't look good. The idea was to fill all width, but it's not too important to have 100% width, so we can make such UX tradeoff

Since this PR consists mostly from changes for that expanding, I'll close it and create new one for renaming metadata -> user metadata